File: app/src/main/java/com/babestudios/companieshouse/data/model/search/Matches.java
package com.babestudios.companieshouse.data.model.search;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class Matches {

	@SerializedName("title")
	public List<Integer> title = new ArrayList<>();

	@SerializedName("snippet")
	public List<Object> snippet = new ArrayList<>();

}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/search/Matches.kt
package com.babestudios.companyinfouk.data.model.search

import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class Matches {

	@SerializedName("title")
	var title: List<Int> = ArrayList()

	@SerializedName("snippet")
	var snippet: List<Any> = ArrayList()

}
